src/core/BufferGeometry.js

```diff
@@ -811,6 +851,60 @@ Object.assign( BufferGeometry.prototype, EventDispatcher.prototype, {
 
 	},
 
+	/**
+	 * Checks for duplicate vertices with hashmap.
+	 * Faces with duplicate vertices are rewired to point to the first instance.
+	 * Unused vertices are not removed.
+	 * @param  {THREE.BufferGeometry} geom Geometry to merge
+	 */
+	mergeVertices: function( geom ) {
```

Comment:
`this` is not referenced in this method.

Can you please explain in more detail what this method does -- and how it is intended to be used?